examples/create_rosa_cluster/create_rosa_sts_cluster/oidc_configuration/oidc_provider/main.tf

```diff
@@ -31,6 +31,10 @@ provider "ocm" {
   url   = var.url
 }
 
+provider "aws" {
+  region = var.cloud_region
```

Comment:
This change is related to another PR I guess
Please remove it

Reply:
removed